src/app/Dashboard/components/DashboardContainer.js

```diff
@@ -104,15 +189,19 @@ export class DashboardContainer extends Component {
           </div>
           <div className='profile-overview col-sm-12'>
             <div className='profile-overview__image' />
-            <MyStatsComponent points={pointsEarned} activities={activitiesLogged} />
-            <SocietyStatsComponent society={society} usedPoints={1508} remainingPoints={326} />
+            <MyStatsComponent
+              points={pointsEarned}
+              activities={activitiesLogged}
+            />
+            <SocietyStatsComponent
+              society={society}
+              usedPoints={1508}
```

Comment:
How come we are still hardcoding these?

Reply:
@Chris-Maina is working on that on his PR